### PR TITLE
Fix the local net indexing with table info change.

### PR DESCRIPTION
**Pull request by larry-aptos**
## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Move Compiler
- [ ] Other (specify)

## Checklist
- [ ] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by larry-aptos** [comment]
* **#14725** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-core/14725?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @larry-aptos and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: crates/aptos/src/node/local_testnet/node.rs [hunk 1]
```diff
@@ -155,7 +155,12 @@ impl NodeManager {
 
         // So long as the indexer relies on storage indexing tables, this must be set
         // for the indexer GRPC stream on the node to work.
-        node_config.storage.enable_indexer = run_txn_stream;
+        // node_config.storage.enable_indexer = run_txn_stream;
+        node_config.indexer_table_info.table_info_service_mode = match run_txn_stream {
+            // Local not should be be responsible for backup or restore of table info tables.
```
Comment (gregnazario) [inline crates/aptos/src/node/local_testnet/node.rs:160]: Sentence is confusing
```suggestion
            // Localnet should be responsible for backup or restore of table info tables.
```

File: crates/aptos/src/node/local_testnet/node.rs [hunk 2]
```diff
@@ -155,7 +155,12 @@ impl NodeManager {
 
         // So long as the indexer relies on storage indexing tables, this must be set
         // for the indexer GRPC stream on the node to work.
-        node_config.storage.enable_indexer = run_txn_stream;
+        // node_config.storage.enable_indexer = run_txn_stream;
```
Comment (gregnazario) [inline crates/aptos/src/node/local_testnet/node.rs:158]: let's remove commented out code lines
```suggestion
        // node_config.storage.enable_indexer = run_txn_stream;
```

File: crates/aptos/src/node/local_testnet/node.rs (hunk 1 above)
Comment (larry-aptos) [reply crates/aptos/src/node/local_testnet/node.rs:160]: Lightspeed!! thanks for catching it... was testing haha 😆 

Going to move back to draft

**Pull request closed without merging**